openshift/s2i/bin/run

```diff
@@ -75,7 +75,7 @@ manage_file=$APP_HOME/manage.py
 if should_migrate; then
   if [[ -f "$manage_file" ]]; then
     echo "---> Migrating database ..."
-    python "$manage_file" migrate --noinput
+    python "$manage_file" migrate
```

Comment:
I'm pretty sure we want the `--noinput` flag because we don't want `migrate` prompting us for things like missing default values. There's nobody at the terminal to respond to such prompting.

Reply:
Yep, adding that guy back in. No clue why this was failing on my local build, but it works again with the flag!